src/simple-pg/filter/filterObject.js

```diff
@@ -0,0 +1,49 @@
+import { unwrapObject, encodePath } from '@graffy/common';
```

Comment:
Are you using `filter` and `link`? I would suggest removing them from the PR for now, bringing them in later when we need them.

Reply:
Was trying to see if I can do something with link and filter but I think they should be in the next version